src/core/tests.py

```diff
@@ -64,3 +65,14 @@ def test_cache_headers_not_in_response(self):
         resp = self.c.get(self.url)
         for header in cases:
             self.assertFalse(resp.has_header(header), "header %r present in response" % header)
+
+    @patch('core.middleware.settings')
```

Comment:
Django has this to patch settings during testing: https://docs.djangoproject.com/en/2.0/topics/testing/tools/#django.test.override_settings

but I think the intent here is just as clear.